rtos/TARGET_CORTEX/mbed_rtx_conf.h

```diff
@@ -80,4 +80,173 @@
 #define OS_IDLE_THREAD_NAME         "idle_thread"
 #define OS_TIMER_THREAD_NAME        "timer_thread"
 
+/* Enable only the evr events we use in Mbed-OS to save flash space. */
+#define EVR_RTX_MEMORY_INIT_DISABLE
+#define EVR_RTX_MEMORY_ALLOC_DISABLE
+#define EVR_RTX_MEMORY_FREE_DISABLE
+#define EVR_RTX_MEMORY_BLOCK_INIT_DISABLE
+#define EVR_RTX_MEMORY_BLOCK_ALLOC_DISABLE
+#define EVR_RTX_MEMORY_BLOCK_FREE_DISABLE
+//#define EVR_RTX_KERNEL_ERROR_DISABLE
```

Comment:
Change looks good. Please remove all commented defines

Reply:
I intentionally left them there as commented to indicate that we use them. Should we add a comment that these are used.